### DISCUSSION TITLE
UI toggle for todo mode

### DISCUSSION THREAD

**Discussion by JeffersonBledsoe** [rxhanson/Rectangle#546] [Ideas]
I've been using todo mode for a bit now and it's really useful! Is there a particular reason it's still hidden behind a terminal toggle rather than having a UI toggle for enabling it? Possibly in the preferences menu?

**Comment by rxhanson** [comment]
In short, I just haven’t gotten around to it. 

There’s a few items that I wanted to take care of before adding it to the ui, like a bug where some users’ todo windows go to unintended locations, and the ability to select which side of the screen the todo window appears on. 

Also, adding things to the ui means that localization placeholders also need to be added - not a big deal, but another task that sometimes bumps this on the back burner. 

Last, I’ve just been working through an injury that’s put me away from the keyboard for a little while. If anyone wants to take a stab at this, they are more than welcome.
